cmd/k8s-operator/sts.go

```diff
@@ -71,13 +72,19 @@ type tailscaleSTSConfig struct {
 
 type tailscaleSTSReconciler struct {
 	client.Client
+	tsnetServer            *tsnet.Server
 	tsClient               tsClient
 	defaultTags            []string
 	operatorNamespace      string
 	proxyImage             string
 	proxyPriorityClassName string
 }
 
+// IsHTTPSEnabledOnTailnet reports whether HTTPS is enabled on the tailnet.
+func (a *tailscaleSTSReconciler) IsHTTPSEnabledOnTailnet(ctx context.Context) bool {
```

Comment:
what's the context for?

Reply:
oh i was using LocalAPI previously, then realized i could just get `CertDomains` will remove